pkg/operator/ceph/cluster/predicate.go

```diff
@@ -123,7 +123,6 @@ func watchControllerPredicate(rookContext *clusterd.Context) predicate.Funcs {
 			switch objOld := e.ObjectOld.(type) {
 			case *cephv1.CephCluster:
 				objNew := e.ObjectNew.(*cephv1.CephCluster)
-				logger.Debug("update event on CephCluster CR")
```

Comment:
Honestly, this _used_ to be useful, was that really a problem? I mean, really verbose?

Reply:
Yes this message is useful, but we get updates of the `update event` from [here](https://github.com/rook/rook/blob/e115e0ada8507a506ddbc8adeb1e6e3bbd6389b5/pkg/operator/ceph/cluster/predicate.go#L119), and this seems to be duplicated information present, is it still required?

Reply:
The intent was to differentiate any update event, versus CephCluster update event? Personally, I'd remove L119 instead and keep this one.